sscanApp/src/saveData_writeXDR.c

```diff
@@ -719,23 +719,32 @@ LOCAL int fileStatus(char* fname)
 {
 	struct stat status;
 	int retVal;
-
+	
```

Comment:
remove the trailing whitespace

Reply:
This whitespace makes the indentation consistent across the entire function (see attached screenshot).

![Screenshot from 2024-07-24 15-02-09](https://github.com/user-attachments/assets/f9a5d60b-0717-4d1e-b8fa-e29877fbfe44)

Reply:
OK